Cargo.toml

```diff
@@ -62,6 +63,7 @@ which = "4.4.0"
 wiremock = "0.5"
 
 [patch.crates-io]
+ethers = { git = "https://github.com/gakonst/ethers-rs", rev = "4acfa9d4e093022aa1bdbe9490e0ffa927494ec1" }
```

Comment:
Why the dependency on upstream?

Ah, found it. This is after this PR got merged in: https://github.com/gakonst/ethers-rs/pull/2644

Can you please add a `TODO` above this patch with a github issue to track removing this once a new ethers release is published?